.github/workflows/windows-validation.yml

```diff
@@ -17,7 +17,7 @@ jobs:
     strategy:
       matrix:
         cache: [false, true]
-        go: [1.20.1]
+        go: [1.21.0]
```

Comment:
This job tests is the link created for non-default go installation and it does not make sense to change it. It is just "any which is not preinsalled in the toolcache"